Electron/src/components/ShowAllItems/types/PropsShowAllItems.ts

```diff
@@ -9,5 +9,5 @@ export enum ShowAllItemsTypes {
 export interface PropsAllItems {
   refreshSidebarData: () => void;
   type: ShowAllItemsTypes;
-  changeSongName: (songName: string) => void;
+  changeSongName?: (songName: string) => void;
```

Comment:
Why do we keep references to the method? Shouldn't we just use the context? I don't think there's a need for that prop anymore. Check the other props modified